### ISSUE TITLE
keyboard ↑↓←→ make nw 0.36.2 crash when enable-features=nw2

### ISSUE THREAD

**Issue by chatcan** [nwjs/nw.js#6976]
Thanks for reporting an issue! Spend 60 seconds reading this...

1. Before reporting, try reproducing against the latest version of NWJS.
2. Issue tracker is only for bugs and feature requests, in English only.
  
  Anything else? Use our mailing list for questions: 
  https://groups.google.com/forum/#!forum/nwjs-general

======== TEMPLATE BUG FORM ========

NWJS Version : 0.36.2
Operating System : Mac 10.14

### Expected behavior

input correctly

### Actual behavior

crash

### How to reproduce

enable-features=nw2
inspect nwjs , focus on the devtools window and press → key

**Comment by chatcan** [comment]
cmd+c, cmd+p, cmd+v cause crash

**Comment by ffanny** [comment]
With nwjs-sdk-win&linux-v0.36.2, it doesn't crash.
And I am not sure if there is a problem with osx-v0.36.2, the mac version.

**Comment by chatcan** [comment]
[test 2.zip](https://github.com/nwjs/nw.js/files/2884170/test.2.zip)

```javascript
const menu = new nw.Menu({type: 'menubar'})
const ideMenu = new nw.Menu()

ideMenu.append(new nw.MenuItem({
  label: 'a',
}))

menu.append(new nw.MenuItem({
  label: 'b',
  submenu: ideMenu,
}))
nw.Window.get().menu = menu
```

package.json

```json
{
  "main":"index.html",
  "name":"cantest",
  "appname":"cantest",
  "window": {
    "width": 800,
    "height": 474,
    "frame": false
  }
}
```

**Comment by Blatman** [comment]
Just tried the test zip and got the problem to occur using 10.13.6 and 0.36.3-sdk and 0.37.0b1-sdk.
Had to add the nw2 switch to package.json of course but fails with Trap 14 error if frame is false -
works OK if frame is true.

Note this issue should be included with #5875 as it is related to nw2 switch.
Cheers.
